matador/scrapers/castep_scrapers.py

```diff
@@ -694,7 +694,10 @@ def param2dict(fname, db=True, **kwargs):
                         while "%endblock devel_code" not in flines[line_no + i].lower():
                             if i + line_no >= len(flines):
                                 raise RuntimeError("Found unclosed %block devel_code.")
-                            line = flines[line_no + i].lower()
+                            if "magres" in flines[line_no + i]:
+                                line = flines[line_no + i].upper()
+                            else:
+                                line = flines[line_no + i].lower()
```

Comment:
I didn't realise the devel codes were case sensitive, so we should really just get rid of the `.lower()` in the first line. Happy to include this for now though as it will probably break some unrelated tests otherwise.

Reply:
I think its just the MAGRES devel that is case sensitive actually...though I can't confirm that and haven't gone through the source to look for it but I agree I think this should be left like this now, so that the tests you have don't fail.

Reply:
Also needed to change 697 to look for `'magres' in flines[line_no + i].lower() `